src/Components/ImagesTable/ImageLink.js

```diff
@@ -74,7 +75,9 @@ const ImageLink = ({ imageId, isExpired, isInClonesTable }) => {
 
   if (
     hasProvisioning &&
-    (image.imageType === 'aws' || image.imageType === 'ami')
+    (image.imageType === 'gcp' ||
```

Comment:
@amirfefer I don't think this we want to expose it to everyone yet, this is not done feature, it needs quite some work to be completed. could we do it only if a feature flag is enabled?

Reply:
I'd vote for removing the gcp for now, just adding the providerType so we are sure Provisioning Wizard does not crash.

Reply:
agree, i removed the gcp for now, adding only `provider`
in a follow-up PR , i'll add a feature flag for gcp provider.